### PR TITLE
module_probe: fix moudle kvm_intel/kvm_amd cannot be unloaded

### PR DESCRIPTION
**Pull request by pingl2016**
Don't start vm for this case.

Signed-off-by: Ping Li <pingl@redhat.com>

### CODE REVIEW

**Comment by pingl2016** [comment]
ID: 1326172

File: generic/tests/module_probe.py [hunk 1]
```diff
@@ -12,6 +12,8 @@ def run(test, params, env):
 
     This tests the kernel pre-installed kernel modules
     """
+    vm = env.get_vm(params["main_vm"])
```
Comment (xutian) [inline generic/tests/module_probe.py]: please use env.get_all_vms() to get VM object from env and destroy it before startup your test

**Comment by xutian** [comment]
Better to add start_vm = no in test configuration.

**Comment by pingl2016** [comment]
@xutian have corrected it.

**Comment by spcui** [comment]
LGTM
Acked-by: scui@redhat.com

**Comment by xutian** [comment]
I think not only need  set start_vm = no but also need to destroy live VM in your test case. maybe other has live VM still running in test loop.

**Comment by pingl2016** [comment]
@xutian have updated the patch. Thanks.

**Pull request merged**